SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialEventTracer.h

```diff
@@ -66,7 +66,12 @@ class SpatialEventTracer
 	void DropSpanIds(const EntityComponentId& EntityComponentId);
 	void DropSpanId(const EntityComponentId& EntityComponentId, const uint32 FieldId);
 
+	static FString SpanIdToString(const Trace_SpanId& SpanId);
```

Comment:
Do we want this public?

Reply:
I just found it useful for debugging. Don't see why it can't be public. Trace_SpanId exist in the code if someone wants to print them they can use this. What be the argument for making it private?

Reply:
Not much, just the usual "prefer private". I can see it being useful.